lib/internal/backend/repository/repository2.go

```diff
@@ -125,7 +125,9 @@ func (rb *RepositoryBackend) buildACIV2(layerIDs []string, dockerURL *types.Pars
 		return nil, nil, err
 	}
 	for _, closer := range closers {
```

Comment:
i) closers can just be `[]io.Closer`
ii) why not only append to `closers` if getLayerV2 returns a nil err?